src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -2616,6 +2618,7 @@ private void readInput() {
     File dockerfileUsed;
     File looseAppFile;
     WatchService watcher;
+    boolean lastChangeCompiled;
```

Comment:
Suggestion to add some comments indicating what lastChangeCompiled is tracking/how it is used.